src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -216,8 +216,9 @@ protected override bool UpdateValueCore(UpdateValueParams updateValueParams)
             return base.UpdateValueCore(updateValueParams);
         }
 
+        [Obsolete]
```

Comment:
obsolete?

Reply:
as a a temporary feature, this will be removed, did this so it raises warnings in the compiler and in VS Intellisense, so anyone using this API will know not to depend on it and that it's already planned to go away.

wanted to ask you about it, let me know if you prefer with the attribute or without.

Reply:
hmm, thats a good point - for now would you mind adding a small message indicating this API will only exist for a short time.

Reply:
done 😄